src/components/learner-credit-management/members-tab/BudgetDetailMembersTabContents.jsx

```diff
@@ -27,16 +27,17 @@ const BudgetDetailMembersTabContents = ({ enterpriseUUID, refresh, setRefresh })
     refresh,
   });
   const {
-    isRemovedMembersLoading,
+    isMembersLoading,
     removedGroupMembersCount,
-  } = useEnterpriseRemovedGroupMembers({
+  } = useEnterpriseGroupMembers({
```

Comment:
[question] i'm curious why we didn't use the existing hook `useEnterpriseGroupLearners` since that hook already returns the learners data so that we can keep the `useEnterpriseRemovedGroupMembers` to just checking for removed members. 

https://github.com/openedx/frontend-app-admin-portal/blob/master/src/components/learner-credit-management/data/hooks/useEnterpriseGroupLearners.js